tester/queue.go

```diff
@@ -2,12 +2,27 @@ package tester
 
 import (
 	"sync"
+
+	"github.com/Shopify/sarama"
 )
 
 type message struct {
-	offset int64
-	key    string
-	value  []byte
+	offset  int64
+	key     string
+	value   []byte
+	headers map[string][]byte
+}
+
+func (m *message) SaramaHeaders() []*sarama.RecordHeader {
```

Comment:
Since it is an exported function, could we add a small comment explaining it?

Reply:
now private and commented